### ISSUE TITLE
Update 1.0.9 crashes setting panel

### ISSUE THREAD

**Issue by rinal13** [rotdrop/nextcloud-app-pdf-downloader#14]
After Update to 1.0.9 the setting panel of Nextcloud crashes with server error.

```
[pdf_downloader] Fatal: Error: Class "OCA\FilesArchive\Constants" not found at <<closure>>

0. /var/www/nextcloud/apps/pdf_downloader/lib/AppInfo/Application.php line 47
   OCA\PdfDownloader\AppInfo\Application->getAppInfoAppName()
1. /var/www/nextcloud/lib/private/ServerContainer.php line 107
   OCA\PdfDownloader\AppInfo\Application->__construct()
2. /var/www/nextcloud/lib/private/ServerContainer.php line 176
   OC\ServerContainer->getAppContainer()
3. /var/www/nextcloud/lib/private/ServerContainer.php line 143
   OC\ServerContainer->getAppContainerForService()
4. /var/www/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php line 129
   OC\ServerContainer->query()
5. /var/www/nextcloud/lib/private/AppFramework/Bootstrap/Coordinator.php line 91
   OC\AppFramework\Bootstrap\Coordinator->registerApps()
6. /var/www/nextcloud/lib/base.php line 661
   OC\AppFramework\Bootstrap\Coordinator->runInitialRegistration()
7. /var/www/nextcloud/lib/base.php line 1111
   OC::init()
8. /var/www/nextcloud/remote.php line 118
   require_once("/var/www/nextcloud/lib/base.php")
```

**Comment by rotdrop** [comment]
Oops. That slipped through because I have the archive-manager also installed and both apps in principle could share code.

I think it should be cured with commit 0f5bf014078e58106e38a47b17c59a864778a9e0

There will be soon a new release.

**Comment by rinal13** [comment]
> Oops. I really should create a test-suite for at least rudimentary testing ... v1.0.10 is broken as well.

can only confirm that it would be a good idea ;)

```
[no app in context] Fehler: Exception: App with id pdf_downloader has invalid signature at <<closure>>

0. /var/www/nextcloud/lib/private/Installer.php line 194
   OC\Installer->downloadApp()
1. /var/www/nextcloud/apps/settings/lib/Controller/AppSettingsController.php line 535
   OC\Installer->updateAppstoreApp("*** sensitive parameters replaced ***")
2. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 225
   OCA\Settings\Controller\AppSettingsController->updateApp("*** sensitive parameters replaced ***")
3. /var/www/nextcloud/lib/private/AppFramework/Http/Dispatcher.php line 133
   OC\AppFramework\Http\Dispatcher->executeController()
4. /var/www/nextcloud/lib/private/AppFramework/App.php line 172
   OC\AppFramework\Http\Dispatcher->dispatch()
5. /var/www/nextcloud/lib/private/Route/Router.php line 298
   OC\AppFramework\App::main()
6. /var/www/nextcloud/lib/base.php line 1030
   OC\Route\Router->match()
7. /var/www/nextcloud/index.php line 36
   OC::handleRequest()`
```

**Comment by rotdrop** [comment]
Should be resolved by release v1.0.11
